libs/parser/src/linkedin/index.ts

```diff
@@ -55,6 +55,9 @@ export class LinkedInParser implements Parser<JSZip, LinkedIn> {
 
   convert(data: LinkedIn) {
     const result = JSON.parse(JSON.stringify(defaultResumeData)) as ResumeData;
+    function avoidToShort(name, len) {
```

Comment:
Nitpick: I think the function name should be `avoidTooShort`

Reply:
Totally true! :+1: